inc/command/class-plugin.php

```diff
@@ -79,4 +79,16 @@ public function install_files() {
 			'https://www.altis-dxp.com/resources/docs/dev-tools/continuous-integration/ ' . "\n"
 		);
 	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function deactivate( Composer $composer, IOInterface $io ) {
```

Comment:
strictly speaking, these probably would be better off in a separate PR unless they are specifically related to the Composer v2 stuff (which I don't think they are?), but I'll allow it.